src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -28,7 +27,7 @@ public partial class MonitoringDescriptionAttribute : System.ComponentModel.Desc
         public MonitoringDescriptionAttribute(string description) { }
         public override string Description { get { throw null; } }
     }
-    public partial class Process : System.ComponentModel.Component
+    public partial class Process : System.ComponentModel.Component, System.IDisposable
```

Comment:
Unrelated to annotation work. Not sure why this was missing here.

Reply:
Because Component itself is IDisposable.